numpy/lib/arraysetops.py

```diff
@@ -343,12 +343,15 @@ def intersect1d(ar1, ar2, assume_unique=False, return_indices=False):
     along with the intersected values:
     
     >>> np.intersect1d([1,2,3,4], [2,1,4,6], return_indices=True)
-    (array(1,2,4), array([0,1,3]), array([1,0,2]))
+    (array([1,2,4]), array([0,1,3]), array([1,0,2]))
+    >>> np.intersect1d([1,2,2,3,4,3,2],[1,8,4,2,2,3,2,3],return_indices=True)
+    (array([1, 2, 3, 4]), array([0, 1, 3, 4]), array([0, 3, 5, 2]))
```

Comment:
These doctests are not run. Can you add some tests to the separate test files?

Reply:
OH yes I wasn't aware of that. I'll add these there quickly.